.npmignore

```diff
@@ -1,3 +1,2 @@
 **/.*
 test
-*.json
```

Comment:
@mirisuzanne This was causing the docs search bug. Search data is stored (and served) as a JSON file, and oddbird.net serves the docs by installing the npm package.